extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -93,7 +93,7 @@ const publicAzureSettings: ProviderSettings = {
 				endpoint: 'https://analysis.windows.net/powerbi/api',
 				azureResourceId: AzureResource.PowerBi
 			},
-			redirectUri: 'https://vscode-redirect.azurewebsites.net/',
+			redirectUri: 'http://localhost',
```

Comment:
Per https://learn.microsoft.com/en-us/azure/active-directory/develop/reply-url#localhost-exceptions we should be using `127.0.0.1` instead of `localhost` - although they also mention you have to do some wonky configuration to do that which seems odd...

Reply:
I saw that too, but I think we'll just use localhost for now, that's also what SSMS is doing